src/ios/THEPeer.m

```diff
@@ -38,7 +38,7 @@ @implementation THEPeer
 }
 
 // Class initializer.
-- (instancetype)initWithIdentifier:(NSUUID *)identifier
+- (instancetype)initWithIdentifier:(NSString *)identifier
                               name:(NSString *)name
```

Comment:
Shouldn't be at least be validating these arguments? Making sure they are not null? Or is that handled upstream? Sorry, still trying to figure my way through all this code.